### DISCUSSION TITLE
Export Project / Resource Configuration

### DISCUSSION THREAD

**Discussion by toooo** [coollabsio/coolify#2248] [Feature Requests]
It would be nice to have a standard way to export configurations similar to digital ocean App spec
https://docs.digitalocean.com/products/app-platform/reference/app-spec/
https://docs.digitalocean.com/glossary/app-spec/

**Comment by platform-kit** [comment]
Agreed. +1

**Comment by ercnshngit** [comment]
+1

**Comment by Cinzya** [comment]
Technically you can already do that via the [Coolify API](https://coolify.io/docs/api-reference/api/operations/get-application-by-uuid). What in specific are you missing that the response from the Coolify API cannot give you?

**Reply by mounirelbakkali1** [reply]
it is not only about describing the resources/configurations. Lunching a project from a configuration file is also highly appreciated feature

**Reply by Cinzya** [reply]
That is also already possible via the Coolify API.

**Comment by discapes** [comment]
I would like to migrate a project from one coolify instance to another. There is a good guide for migrating the docker volumes, but no mention of how to migrate the project configuration, instead of having to recreate it manually by hand.

**Comment by mahiuddinalkamal** [comment]
+1
